src/PaymentMethods/PaymentFieldsStrategies/BillieFieldsStrategy.php

```diff
@@ -6,7 +6,7 @@
 
 class BillieFieldsStrategy implements PaymentFieldsStrategyI
 {
-    const FIELD_COMPANY = "billing_company";
+    const FIELD_COMPANY = "billing_company_billie";
```

Comment:
It is recommended to use the single quotes for the simple strings, also can add the visibility to constant.

```suggestion
    public const FIELD_COMPANY = 'billing_company_billie';
```